app/src/components/organisms/SurveyClassQuestion.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import Stack from '@mui/material/Stack';
+import { willingRadio, ability, willing } from '../shared/constants/surveyForm.constants';

```

Comment:
Let's change this relative import to an absolute one ('components/shared/constants/surveyForm.constants';